src/whylogs/core/columnprofile.py

```diff
@@ -111,9 +111,7 @@ def track(self, value, character_list=None, token_method=None):
 
             self.string_tracker.update(value, character_list=character_list, token_method=token_method)
 
-        # TODO: Implement real typed data conversion
-        typed_data = TypedDataConverter.convert(value)
-
+        typed_data = value
```

Comment:
The TypedDataConverter seems to take a string object and use the yaml parser to convert it to a more appropriate type. I know Isaac had discussed getting rid of this as a default, but was that your goal here?

My understanding is that this change affects the underlying profiling, not just the summary.

Reply:
Good catch. This seems to be a big change to the behavior

Reply:
@andyndang isnt this the behaviour in the java side ?

Reply:
the idea is that string are strings and int are ints. parse ints from string is not the behavours that most users want, for example " 00123" is usually expected as a string for so that it can keep track of the leading zeros, think upcs. Once it is converted as a feature then it is consumed into own format, as a string , int or encoding as the model user defines it.

Reply:
Yes currently it is. We are trying to do type-detection. This is a significant behavior change and I would not include it here

Reply:
moving to a seperate PR

Reply:
done